src/client/app/utils/dateRangeCompatibility.ts

```diff
@@ -46,7 +52,14 @@ export function dateRangeToTimeInterval(dateRange: Value): TimeInterval {
 	}
 	if (start && end) {
 		start = moment(toUTC(start));
-		end = moment(toUTC(end));
+		/* Adds a millisecond to the end time.
+		For the case in which the end time is the last moment of the day (date = date.endOf('day)),
+		adding a millisecond pushes the end time to the very beginning of the next day.
+		i.e the end date is 7.22.2020 at 23:59:59. Due to how OED works, the end date will be truncated to
+		7.20.2020 at 00:00:00. This is due to OED decision to graph only entire days causing the lost of the last date.
```

Comment:
I think 7.20 -> 7.22. Also, if you want to use the standard representation that the code uses then it is 2020-07-22 23:59:59 (as example).